### ISSUE TITLE
trailing whitespace on empty lines does not trigger rule

### ISSUE THREAD

**Issue by echorink** [clutchski/coffeelint#39]
having whitespace on an **empty** line does not trigger the trailing-whitespace rule, even though leading whitespace might exist due to autoindenters...  might be nice to have a separate rule for this corner case?

``` coffeescript
    # blahblah
    # next line contains trailing whitespace (really, though markdown strips it!)

EOF
```

was expecting it to trigger "Line 3 contains trailing whitespace"

**Comment by vaiwa** [comment]
Please add parameter 'allowed_in_empty_lines' to the documentation at http://www.coffeelint.org/
